feature/gnmi/otg_tests/telemetry_interface_packet_counters_test/telemetry_interface_packet_counters_test.go

```diff
@@ -240,6 +250,18 @@ func TestIntfCounterUpdate(t *testing.T) {
 		// "ipv4":   subintf2.Ipv4().Counters().OutPkts().Get(t),
 		// "ipv6":   subintf2.Ipv6().Counters().OutPkts().Get(t),
 	}
+	if *deviations.InterfaceCountersFromContainer {
+		dutInPktsBeforeTraffic = map[string]uint64{
+			"parent": *gnmi.Get(t, dut, i1.Counters().State()).InUnicastPkts,
+			// "ipv4":   *subintf1.Ipv4().Counters().Get(t).InPkts,
```

Comment:
remove lines 256,257 and 261,262 if not relevant

Reply:
These are associated with the TODO at line 238 (https://github.com/openconfig/featureprofiles/pull/1415/files#diff-9c42efab3f06ea8f988e62604a01384d33c1c2f79bc98a13d4dcf67543053a7cR238), which also applies to the ATE test that also has these lines commented out.

I'd err on the side of keeping the extra noise here to make reviews of updates that apply for both tests easier.